styles.css

```diff
@@ -102,3 +153,138 @@
 .fa:hover {
   color: #36b37e;
 }
+
+.grid-container {
+  padding-top: 50px;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+}
+
+.grid-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 50px 30px 50px 30px;
+  padding-left: 10px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.work-section-image {
+  margin: 10px 10px 10px 1px;
+}
+
+.client-info {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.client-name {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.project-style .project-year {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.client-section-counter {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.project-info {
+  font-family: Poppins, sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 5px;
+}
+
+.language-used {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: 12px 5px;
+}
+
+.tags {
+  background: #ebebff;
+  color: #6070ff;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 1px 10px;
+  padding: 5px 15px 5px 15px;
+}
+
+.button {
+  background: #fff;
+  flex: none;
+  position: static;
+  width: 124px;
+  height: 48px;
+  left: 1px;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  order: 0;
+  flex-grow: 0;
+  margin: 1px 10px 10px 10px;
+}
+
+.button:hover {
+  color: white;
+  background: #6070ff;
+}
```

Comment:
- [ ] Good job changing the properties of the buttons with `:hover` state. We can do the same to change the `:active` state which will render when the user clicks the button. 

![image](https://user-images.githubusercontent.com/79114233/138366434-e5c51f35-8039-496e-b905-c70350adcc7b.png)